pkg/model/iam/iam_builder.go

```diff
@@ -780,6 +783,19 @@ func AddAWSLoadbalancerControllerPermissions(p *Policy, resource stringorslice.S
 	)
 }
 
+func addSnapshotPersmissions(p *Policy) {
+	p.Statement = append(p.Statement, &Statement{
+		Effect: StatementEffectAllow,
+		Action: stringorslice.Of(
+			"ec2:CreateSnapshot",
+			"ec2:DeleteSnapshot",
+			"ec2:DescribeAvailabilityZones",
+			"ec2:DescribeSnapshots",
+		),
+		Resource: stringorslice.Slice([]string{"*"}),
```

Comment:
Can we restrict this any further for ec2:DeleteSnapshot ? It seems pretty broad as-is.

Reply:
Yeah. This one got lost as well. I'll fix this shortly.